README.md

```diff
@@ -47,6 +47,7 @@ You will need a plugin manager to install the plugins and prompts listed on this
 
 - [Official Documentation](http://fishshell.com/docs/current/index.html)
 - [fish cookbook](https://github.com/brj/fish-cookbook) - Tips and recipes for fish-shell users
+- [fish Tutorial](https://github.com/ellerbrock/tutorial-fish-shell-setup-osx) - How to Setup fish Shell with fisherman, Powerline Fonts, iTerm2 and Budspencer Theme on OS X.  
```

Comment:
so you are one of the creators, thats cool man! spending so many hours in the shell day by day and now in contact with one of their fathers - thats awesome :) really nice job, love the fish shell! when i got more time i will extend the list with features and plugins. since i was just reinstalling my laptop i also was thinking just to publish a small script for beginners to set them up with all the nice things like common line fonts, and so on ... hm, just writing this. i guess i write this script now and update my doc :) by the way if i saw it right you are in japan, cheers here from very hot taiwan ...